code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -282,11 +283,182 @@
+	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 10, bio = 0, rad = 0)
+	hoodtype = /obj/item/clothing/head/winterhood/science
+
+/obj/item/clothing/head/winterhood/science
+	icon_state = "winterhood_sci"
+
+/obj/item/clothing/suit/hooded/wintercoat/engineering
+	name = "engineering winter coat"
+	icon_state = "wintercoat_engi"
+	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 0, rad = 20)
+	allowed = list(/obj/item/device/flashlight, /obj/item/weapon/tank/emergency_oxygen, /obj/item/device/t_scanner, /obj/item/weapon/rcd)
+	hoodtype = /obj/item/clothing/head/winterhood/engineering
+
+/obj/item/clothing/head/winterhood/engineering
+	icon_state = "winterhood_engi"
+
+/obj/item/clothing/suit/hooded/wintercoat/engineering/atmos
+	name = "atmospherics winter coat"
+	icon_state = "wintercoat_atmos"
+	hoodtype = /obj/item/clothing/head/winterhood/engineering/atmos
+
+/obj/item/clothing/head/winterhood/engineering/atmos
+	icon_state = "winterhood_atmos"
+
+/obj/item/clothing/suit/hooded/wintercoat/hydro
+	name = "hydroponics winter coat"
+	icon_state = "wintercoat_hydro"
+	allowed = list(/obj/item/weapon/reagent_containers/spray, /obj/item/device/analyzer/plant_analyzer, /obj/item/seeds, /obj/item/weapon/reagent_containers/glass/bottle, /obj/item/weapon/hatchet, /obj/item/weapon/storage/bag/plants)
+	hoodtype = /obj/item/clothing/head/winterhood/hydro
+
+/obj/item/clothing/head/winterhood/hydro
+	item_state = "winterhood_hydro"
+
+/obj/item/clothing/suit/hooded/wintercoat/cargo
+	name = "cargo winter coat"
+	icon_state = "wintercoat_cargo"
+	hoodtype = /obj/item/clothing/head/winterhood/cargo
+
+/obj/item/clothing/head/winterhood/cargo
+	icon_state = "winterhood_cargo"
+
+/obj/item/clothing/suit/hooded/wintercoat/miner
+	name = "mining winter coat"
+	icon_state = "wintercoat_miner"
+	allowed = list(/obj/item/weapon/pickaxe, /obj/item/device/flashlight, /obj/item/weapon/tank/emergency_oxygen, /obj/item/toy, /obj/item/weapon/storage/fancy/cigarettes, /obj/item/weapon/lighter)
+	armor = list(melee = 10, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 0, rad = 0)
+	hoodtype = /obj/item/clothing/head/winterhood/miner
+
+/obj/item/clothing/head/winterhood/miner
+	icon_state = "winterhood_miner"
+
 
 /*
  * Misc
  */
 
+//hoodies
+/obj/item/clothing/suit/hooded/hoodie
+	name = "hoodie"
+	desc = "It's a hoodie. It has a hood. Most hoodies do."
+	icon_state = "black_hoodie"
+	item_state = "labcoat"
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
+	hoodtype = /obj/item/clothing/head/hood
+
+/obj/item/clothing/head/hood
+	name = "hood"
+	desc = "A hood attached to a hoodie."
+	icon_state = "blackhood"
+	body_parts_covered = HEAD
+	cold_protection = HEAD
+	flags = NODROP|BLOCKHAIR
+	flags_inv = HIDEEARS

```

Comment:
Can add the Vox species fitting stuff and remove the species fitting from children here too once once the new sprites are added